makefile

```diff
@@ -19,12 +19,12 @@ format:
 
 lint:
 	python -m flake8 $(PKG)
+	python -m mypy -p $(PKG)
```

Comment:
I think that mypy checks are suitable for this section since type checks are part of linting stage.

Reply:
I disagree with this, because mypy is testing the type annotations and finding errors in code, rather than just linting for best practices.